harness/determined/exec/gc_checkpoints.py

```diff
@@ -155,11 +161,12 @@ def main(argv: List[str]) -> None:
 
     manager = storage.build(storage_config, container_path=constants.SHARED_FS_CONTAINER_PATH)
 
-    if len(storage_ids) > 0:
+    if len(storage_ids) > 0 and storage_ids != [""]:
```

Comment:
do we know what on master side is causing storage_ids to be ```[""]```? Is it possible to fix that?

Reply:
yup. in `runcheckpointGCTask` in `checkpoint_gc.go`, we don't ignore an empty checkpoint list. If `deleteTensorboards` is set, we set the gc spec to an empty list, which is converted to an empty string "" and then a list with an empty string in `gc_checkpoints.py`:
```
// in checkpoint_gc.go
gcSpec := tasks.GCCkptSpec{
		Base:               taskSpec,
		ExperimentID:       expID,
		LegacyConfig:       legacyConfig,
		ToDelete:           deleteCheckpointsStr,
		CheckpointGlobs:    checkpointGlobs,
		DeleteTensorboards: deleteTensorboards,
	}

// in gc_checkpoints.py
storage_ids = [s.strip() for s in args.delete] // where args.delete is set by ToDelete above.
```
As to fixing it, this is the solution I came up with since I don't think we can get around the string -> list  conversion.